forte/processors/base/batch_processor.py

```diff
@@ -252,6 +257,116 @@ def define_batcher() -> ProcessingBatcher:
         return FixedSizeDataPackBatcher()
 
 
+class Predictor(BaseBatchProcessor):
+    @staticmethod
+    def _define_context() -> Type[Annotation]:
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        return Sentence
+
+    @staticmethod
+    def _define_input_info() -> DataRequest:
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        return {}
+
+    def _prepare_coverage_index(self, input_pack: DataPack):
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        pass
+
+    def pack(self, pack: PackType, inputs) -> None:
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        pass
+
+    @staticmethod
+    def define_batcher() -> ProcessingBatcher:
+        return FixedSizeDataPackBatcherWithExtractor()
+
+    @classmethod
+    def default_configs(cls) -> Dict[str, Any]:
+        super_config = super().default_configs()
+        super_config.update({
+            "scope": None,
+            "feature_scheme": None,
+            "batch_size": None,
+            "model": None,
+            "batcher": cls.define_batcher().default_configs()
+        })
+        return super_config
+
+    def initialize(self, resources: Resources, configs: Optional[Config]):
+        batcher_config = {}
```

Comment:
It is unclear of the types of these class varaibles.

Reply:
add type.